routers/install.go

```diff
@@ -60,6 +60,7 @@ func Install(ctx *context.Context) {
 	form.DbUser = models.DbCfg.User
 	form.DbName = models.DbCfg.Name
 	form.DbPath = models.DbCfg.Path
+	form.DbPasswd = models.DbCfg.Passwd
```

Comment:
Please move the line below `form.DbUser` to reflect the actual order of the install page form fields.